packages/@react-types/color/src/index.d.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {BaseSliderProps} from '@react-types/slider';
+import {Color} from '@react-stately/color';
+import {DimensionValue, StyleProps, ValueBase} from '@react-types/shared';
+import {HTMLAttributes, InputHTMLAttributes, RefObject} from 'react';
+
+/** A list of supported color formats. */
+export type ColorFormat = 'hex' | 'hexa' | 'rgb' | 'rgba' | 'hsl' | 'hsla' | 'hsb' | 'hsba';
+
+/** A list of color channels. */
+export type ColorChannel = 'hue' | 'saturation' | 'brightness' | 'lightness' | 'red' | 'green' | 'blue' | 'alpha';
+
+export interface ColorWheelState {
```

Comment:
And usually keep the state interface in the stately package since everything that uses it depends on it